velox/common/memory/MemoryArbitrator.h

```diff
@@ -170,9 +170,31 @@ class MemoryArbitrator {
     /// The free memory capacity in bytes.
     uint64_t freeCapacityBytes{0};
 
+    Stats(
+        uint64_t _numRequests,
+        uint64_t _numAborted,
+        uint64_t _numFailures,
+        uint64_t _queueTimeUs,
+        uint64_t _arbitrationTimeUs,
+        uint64_t _numShrunkBytes,
+        uint64_t _numReclaimedBytes,
+        uint64_t _maxCapacityBytes,
+        uint64_t _freeCapacityBytes);
+
+    Stats() = default;
+
+    Stats operator-(const Stats& other) const;
```

Comment:
Can you comment about the special treatment on max capacity and free capacity? Thanks!